validator/client/validator.go

```diff
@@ -749,6 +752,31 @@ func (v *validator) RolesAt(ctx context.Context, slot primitives.Slot) (map[[fie
 		copy(pubKey[:], duty.PublicKey)
 		rolesAt[pubKey] = roles
 	}
+
+	aggregator, err := v.isSyncCommitteeAggregator(
+		ctx,
+		slot,
+		syncCommitteeValidators,
+	)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "could not check if validators are a sync committee aggregator")
```

Comment:
This should be an error log. I guess something like this will work?
```suggestion
        log.WithError(err).Error("Could not check if any validator is a sync committee aggregator")
		return rolesAt, nil
```

Reply:
D'oh, right! I'll refactor this as an error.